docs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nikolaik/python-nodejs:python3.8-nodejs18
+FROM nikolaik/python-nodejs:python3.9-nodejs18
```

Comment:
could make sense to replace it with 

```suggestion
FROM node:18
```

cc @regisss

Reply:
If the workflow works with this image, sure let's do it!

Reply:
merging to fix main broken since https://github.com/huggingface/optimum/pull/2086, will open a following PR